data/bundle.go

```diff
@@ -109,6 +109,7 @@ type BundleCommand struct {
 // BundleKubernetes represents the "bundles/kubernetes" subsection of the config doc
 type BundleKubernetes struct {
 	ServiceAccountName string `yaml:"serviceAccountName,omitempty" json:"serviceAccountName,omitempty"`
+	EnvSecret          string `yaml:"env_secret,omitempty" json:"env_secret,omitempty"`
```

Comment:
Can `env_secret` be camelCased?

Reply:
Actually: scratch that. It looks like all of the existing code is inconsistent with with the JSON standard, so proceed and I'll fix the entire codebase another time. :) 

Carry on. Nothing to see here.